theme/lumo/vaadin-avatar-styles.html

```diff
@@ -31,21 +31,24 @@
 
       [part="abbr"] {
         font-family: var(--lumo-font-family);
-        font-size: 3em;
+        font-size: 2.4375em;
         font-weight: 500;
       }
 
-      :host([theme~="xlarge"]) [part="abbr"],
+      :host([theme~="xlarge"]) [part="abbr"] {
+        font-size: 2.75em;
```

Comment:
Looks like `2.5em` would be better. Here is a pixel perfect match:

<img width="395" alt="Screenshot 2020-08-18 at 14 08 59" src="https://user-images.githubusercontent.com/10589913/90506148-7ab2cb00-e15c-11ea-99a1-7e8277d4c341.png">

Reply:
Good point! Updated.